docker/bin/prep_demo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+DEIS_BIN="${DEIS_BIN:-deis}"
+
+echo "Creating the demo app $DEIS_APPLICATION"
+$DEIS_BIN apps:create "$DEIS_APPLICATION" --no-remote || true
+
+echo "Configuring the new demo app"
+
+ENV_FILES=(
+  "jenkins/branches/demo/default.env"
+  "jenkins/regions/${DEIS_PROFILE}.env"
+  "jenkins/branches/${BRANCH_NAME}.env"
+)
+
+# reads which ever of the above files exist in order and combines values
+# pre-installed in jenkins
+ENV_VALUES=( $(envcat "${ENV_FILES[@]}") )
```

Comment:
That's fine for a short term solution, but would you mind filing an issue (or even better a PR) against https://github.com/mozmar/ee-infra-jenkins/ to add envcat?